desktop/src/types.ts

```diff
@@ -206,7 +206,11 @@ export type TImportWorkspaceConfig = Readonly<{
 //#region Context
 export type TContextOptions = Record<TContextOptionName, TContextOption>
 // See pkg/config/context.go
-export type TContextOptionName = "AGENT_URL" | "TELEMETRY"
+export type TContextOptionName =
+  | "AGENT_URL"
```

Comment:
do you want the first `|`?

Reply:
That's prettiers auto formatting, not going to fight it :D